examples/with-gatsby/package.json

```
@@ -0,0 +1,25 @@
+{
+  "private": true,
+  "name": "with-gatsby",
+  "license": "MIT",
+  "version": "0.4.2-canary.2",
+  "scripts": {
+    "start": "gatsby develop",
+    "develop": "gatsby develop",
+    "build": "gatsby build",
```

Optional: add a lint rule here.

```suggestion
    "build": "gatsby build",
    "lint": "tslint --project tsconfig.lint.json --config ../../tslint.json",
```